packages/inter-protocol/src/proposals/upgrade-vaults.js

```diff
@@ -52,8 +55,20 @@ export const upgradeVaults = async (
   produceVaultInstallation.reset();
   produceVaultInstallation.resolve(installationP);
 
-  await auctionsUpgradeComplete;
-  auctionsUpgradeCompleteProducer.reset();
+  const [auctionOldInstance, auctionNewInstance] = await Promise.all([
+    auctioneerInstanceP,
+    auctionUpgradeNewInstance,
+  ]);
+  auctionOldInstance !== auctionNewInstance ||
+    Fail`Auction instance didn't change`;
+  auctionUpgradeNewInstanceProducer.reset();
+  const publicFacet = E(zoe).getPublicFacet(auctionNewInstance);
+  /** @type {import('@agoric/inter-protocol/src/auction/scheduler.js').FullSchedule} */
+  const schedules = await E(publicFacet).getSchedules();
+  const now = await E(chainTimerService).getCurrentTimestamp();
+  (schedules.nextAuctionSchedule &&
+    TimeMath.compareAbs(schedules.nextAuctionSchedule.startTime, now) > 0) ||
+    Fail`Expected next start time in the future ${schedules.nextAuctionSchedule?.startTime}`;
```

Comment:
I thought we'd log the schedule and eyeball it, but having the computer verify that it's fixed is perhaps more relevant.